<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Astrum Audioplayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.199, y1:0.772, x2:1, y2:0, stop:0 rgba(98, 19, 255, 255), stop:1 rgba(255, 255, 255, 255));
border: 10px;
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="add">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic_fluent_add_square_24_filled.png</normaloff>ic_fluent_add_square_24_filled.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="musicList">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>511</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 10px;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="recordPlayer" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid;
border-color: rgba(98, 19, 255, 255);
border-radius: 10px;
background-color: qlineargradient(spread:reflect, x1:0.199, y1:0.772, x2:1, y2:0, stop:0 rgba(98, 19, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QToolButton" name="prev">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ic_fluent_previous_24_filled.png</normaloff>ic_fluent_previous_24_filled.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="play">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ic_fluent_play_24_filled.png</normaloff>ic_fluent_play_24_filled.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="stop">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ic_fluent_stop_24_filled.png</normaloff>ic_fluent_stop_24_filled.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="next">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ic_fluent_next_24_filled.png</normaloff>ic_fluent_next_24_filled.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="temporaryTrack">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="sliderPosition">
      <number>9</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="audioVolume">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>31</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border: None;</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="musicCover">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="musicName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="closeWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ic_fluent_pane_close_24_filled.png</normaloff>ic_fluent_pane_close_24_filled.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLCDNumber" name="timer">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;
background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
